submissions/JuliaNovak/html-css-popup/styles.css

```diff
@@ -0,0 +1,199 @@
+:root {
+    --hover-color: rgb(103, 103, 200);
+    --bg-color:rgb(245, 243, 243);
+    --secondary-bg-color: 	rgb(211,211,211);
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.menu {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    padding: 0 1rem;
+    background-color: rgb(173,216,230);
+}
+
+.nav-items-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu-item{
+    display: inline-block;
+    margin-right: .6rem;
+}
+
+.special-item {
+    margin-right: 1.9rem;
+}
+
+.menu img{
+    max-width: 25px;
+    padding: 0px;
+}
+
+li {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+a:hover, a:focus, .popup-item img:hover, .popup-item:focus {
+    color: var(--hover-color);
+    outline: 2px solid var(--hover-color);
+    border-radius: 5px;
+}
+
+.popup-item, .popup-label, .input-more {
+    cursor: pointer;
+    position: absolute;
+    /* width: 37px; */
+    top: 1rem;
+    right: 7.7rem;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+}
+
+.dropdown {
+    position: absolute;
+    max-width: 18.75rem;
+    max-height: 21.8rem;
+    border: 2px solid var(--secondary-bg-color);
+    right: 7.5rem;
+    top: 4.6rem;
+    background-color: var(--bg-color); 
+    box-shadow: 0px 0px 15px 5px rgba(0, 0, 0, 0.1); 
+    overflow: hidden auto;
+    display: none;
+    padding: 1rem;
+}
+
+input.popup-item {
+    width: 2.2rem;
+    height: 2.1rem;
+}
+
+input.popup-item:checked ~ .dropdown{
+    display: block;
+}
+
+.wrapper{
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(4, 1fr);
+    margin-bottom: 1.5rem;
+    padding: 0;
+}
+
+img {
```

Comment:
please avoid styling with tags
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css